src/components/AboutProton/AboutProton.scss

```diff
@@ -0,0 +1,47 @@
+
+.AboutProton {
+  @include breakpoint($breakpoint-small) {
+    flex-direction: column;
+  }
+}
+.About {
+  width: 50%;
+  height: 768px;
+  background: #272b35 0% 0% no-repeat padding-box;
+  @include breakpoint($breakpoint-small) {
+    // background-image: url("~../../../../pictures/mountain\ overlay.jpg");
+    width: 100%;
+  }
+}
+.AboutTitle {
+  margin-bottom: 10%;
+  text-align: left;
+  font: normal normal bold 59px/57px Arial;
+  letter-spacing: 0px;
+  opacity: 1;
+  color: #ffffff;
+  @include breakpoint($breakpoint-small) {
```

Comment:
You have responsive helpers to do this, please have a look at https://design-system.protontech.ch/responsive-helpers/